tests/test_photo_ids.py

```diff
@@ -33,3 +33,42 @@ def test_get_photo_by_item_id_photo_not_exist(client: TestClient):
     assert response.json() == {
         "detail": "Item not found"
     }
+
+
+def test_add_photos_by_item_id(client: TestClient):
+    item_id = "0df1dacb-67f6-495c-b993-49d06a293787"
+    json = ["003791b5-6019-4144-b174-9aaaf8095063", "004591b5-6019-4144-b174-9aaaf8095063"]
+    response = client.post(f"/api/items/{item_id}/photos", json=json)
+
+    assert response.status_code == 200
```

Comment:
```suggestion
    assert response.status_code == 201
```

新增成功回傳的狀態碼要是 201，需求打錯了，issue 那邊有更新了